AsyncDisplayKit/ASTableView.mm

```diff
@@ -675,6 +703,19 @@ - (void)tableView:(UITableView *)tableView didEndDisplayingCell:(_ASTableViewCel
 #pragma mark - 
 #pragma mark Batch Fetching
 
+- (void)_checkForBatchFetching
+{
+  // Dragging will be handled in scrollViewWillEndDragging:withVelocity:targetContentOffset:
+  if ([self isDragging] || [self isTracking] || ![self _shouldBatchFetch]) {
```

Comment:
Regarding `isDragging` and `isTracking` see my answer to your comment a bit later.

A new function for combining logic around batch fetching for ASCollectionView and ASTableView was created with latest commit.